meta/meta.go

```diff
@@ -506,9 +509,16 @@ func (m *Meta) getDDLJob(key []byte, index int64) (*model.Job, error) {
 }
 
 // GetDDLJob returns the DDL job with index.
-func (m *Meta) GetDDLJob(index int64) (*model.Job, error) {
+// If the length of jobListKeys isn't zero, we need to replace m.jobListKey with jobListKeys[0].
+// Otherwise, we use m.jobListKey directly.
+func (m *Meta) GetDDLJob(index int64, jobListKeys ...JobListKeyType) (*model.Job, error) {
```

Comment:
From the usage, I can tell there will be always one and only one jobListKey. So why we need a slices here?